ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+#~~~~~~~~#
+# Garula #
+#~~~~~~~~#
+
+# -ic "Steppe Coeurl" "Steppe Sheep" "Steppe Yamaa"
+
+0 "Start" sync / 00:0839:Bardam's Hunt will be sealed off/
+11.7 "Heave" sync /:Garula:1EF7:/ window 11.7,5
+20.9 "Crumbling Crust (cast)" sync /:Garula:1EF8:/
+22.4 "Crumbling Crust (circles)" sync /:Garula:1F13:/
```

Comment:
I think usually I try to pick one of these and either skip or `--sync--` the other.  My usual strategy is if it's something you're mitigating, to pick the cast (because that's the snapshot on the boss) or for this maybe picking the circles would be reasonable.

Reply:
That's fair. I was more looking at this from the perspective of the user, where "The cast went off but the timeline hasn't got there yet!". But in this situation since it's followed up by under 3 seconds it's probably fine, yes.